test/videojs-hls_test.js

```diff
@@ -2651,4 +2651,19 @@ test('does not download segments if preload option set to none', function() {
   strictEqual(loadedSegments, 0, 'did not download any segments');
 });
 
+test('pts value undefined test', function(){
+	//Making a fictious array
+    var tags = [{ pts: 1000, bytes: new Uint8Array(1)}];
+	//Check the length of tags
+	ok(tags.length,"We have the length of tags");
+	//Check to see if value of pts exists
+	ok(tags[length].pts,"We have got a pts value at length of tag");
+	//Check to see if the last element is not undefined
+	notEqual(tags[length].pts,undefined,"the tags array is doing fine until the last pts value");
+	//Check to see if the out of bounds array is undefined
+	equal(tags[length+1],undefined,"the tags array will going out of bounds here");
```

Comment:
This test doesn't exercise any of contrib-hls. You want to show that when seeking, we can iterate off the end of the parsed tags if the target PTS value is greater than the PTS value of the latest tag. You'll probably need to involve the [mockSegmentParser](https://github.com/vdeshpande/videojs-contrib-hls/blob/pull-request-vineet/test/videojs-hls_test.js#L1479) and an instance of the HLS tech itself.